source/katana/install.md

```diff
@@ -0,0 +1,48 @@
+---
+title: sabre/katana
+product: katana
+layout: default
+---
+
+If you downloaded sabre/katana as an archive, skip the pre-requisites.
```

Comment:
Downloading the zip should be the _only_ supported installation method. We can talk about composer and bower in a separate section specifically for people _developing_ katana.